fmriprep/workflows/bold/base.py

```diff
@@ -905,18 +904,6 @@ def init_func_preproc_wf(bold_file):
     return workflow
 
 
-def _get_series_len(bold_fname, skip_vols):
-    from niworkflows.interfaces.registration import _get_vols_to_discard
-    img = nb.load(bold_fname)
-    if len(img.shape) < 4:
-        return 1
-
-    if skip_vols is None:
-        skip_vols = _get_vols_to_discard(img)
```

Comment:
We lose the case where the series has >5  volumes but <5 taking into account dummy scans.

Reply:
Yes, this is what I meant (and definitely failed to convey) above.